#  Message describing a set of stamped trajectories

Vec3Trajectory[] trajectories
